iothub/device/src/Transport/AmqpClientConnection.cs

```diff
@@ -37,7 +37,7 @@ internal abstract class AmqpClientConnection
 
         internal TlsTransportSettings tlsTransportSettings { get; private set; }
 
-        public AmqpConnection amqpConnection { get; protected set; }
+        public AmqpConnection AmqpConnection { get; protected set; }
 
```

Comment:
I don't follow this coding standard. Shouldn't it be camelCase for member name?

Reply:
I'm not sure. There are a lot of classes using upper case, like `MethodInvokeRequest`, `MethodRequest` etc.

Reply:
Property naming should follow PascalCasing